composer-dependencies.json

```diff
@@ -0,0 +1,13 @@
+{
+    "require": {
+        "krokedil/klarna-onsite-messaging": "^1.0.0"
+    },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:krokedil/klarna-onsite-messaging.git"
+        }
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true
```

Comment:
You are also missing the configuration requirement here for the PHP version. You should add that here to prevent issues with building it for your own local PHP version, which would probably be a later version than we support.

So add in this block to force it to build for PHP 7.4+ only.
```json
"config": {
        "platform": {
            "php": "7.4"
        }
    }
```